packages/manager/modules/cloud-connect/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud CloudConnect product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.14"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^9.0.0 || ^10.0.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.5.0",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "bootstrap": "^4.3.1",
+    "moment": "^2.24.0",
```

Comment:
Why not add them as dependencies ? Also it seems those peers are missing from the `cloud-connect-app` dependencies